portfolio/src/main/java/com/google/sps/servlets/Week2Servlet.java

```diff
@@ -0,0 +1,35 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/week-2")
+public class Week2Servlet extends HttpServlet {
+
+	List<String> messages = new ArrayList<String>();
+
+	@Override
+	public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+		// Add hardcoded messages to ArrayList
+		messages.add("Maybe you received this message first");
+		messages.add("Maybe you received this message second");
+		messages.add("Maybe you received this message third");
+		messages.add("Maybe you received this message last");
+		messages.add("Maybe you received no messages at all");
```

Comment:
I think this will add more messages to the list every time a GET request is made. If the messages are constant and never change, it may be better to initialize the list above with values, for example

List<String> messages = Arrays.asList("message 1", "message 2")

Reply:
Changed messages to Arrays.asList() with hard-coded strings instead of adding strings to empty list